base/exports.jl

```diff
@@ -632,6 +634,7 @@ export
     rpad,
     rsplit,
     rstrip,
+    rtrunc,
```

Comment:
@nsajko Regarding renaming. I'm wavering. We have `rpad` `rsplit` `rstrip`, `lpad` `lstrip`.
Moving to `truncate_right` etc. feels a bit awkward for something so closely tied.

What about `rtruncate`?

Reply:
Yeah, `rtruncate` is much better, I vote for `rtruncate`.

Reply:
I do generally like unabbreviated names (`nextind` continues to annoy me).

However... I do wonder if there's a danger in `ltruncate` (& co.) being very similar in name to `truncate` yet doing something completely different?